api/factory/gin/common.go

```diff
@@ -1,10 +1,11 @@
-package ginwebserver
+package gin
```

Comment:
Is there a reason for the gin wrapper implementation to be part of the `factory` package? If you search through elrond-go repo you will find that factory packages usually contain the struct + unit test file that creates the implementation based on some arguments and configs (using a switch statement). I would prefer moving this gin folder outside of the factory package.

Reply:
done. moved